workers.tf

```diff
@@ -126,7 +126,15 @@ resource "aws_autoscaling_group" "workers" {
           "propagate_at_launch" = true
         },
       ],
-      local.asg_tags,
+      [
+        for item in keys(var.tags) :
+        map(
+          "key", item,
+          "value", element(values(var.tags), index(keys(var.tags), item)),
+          "propagate_at_launch", "true"
+        )
```

Comment:
It's probably because you take it from the `local.asg_tags`, but can't we simplify this writing ?

```hcl
for k,v in var.tags :
map(
  "key", k,
  "value", v,
  "propagate_at_launch", "true"
)
```
Same point in worker_groups_launch_template ASG.

Reply:
Hey @barryib good suggestion. I had thought about that but wanted to keep my first PR fairly as simple as possible. I've made these changes. 

Billy